app/Ammunition/armours.py

```diff
@@ -0,0 +1,8 @@
+class Armour:
+
+    armours = {}

```

Comment:
Why do you need to store armours in class attribute? Create Armour instances and add it to Knight instantly as it gets created